docs/source/release/v6.3.0/framework.rst

```diff
@@ -36,6 +36,7 @@ Bugfixes
 ########
 - Fixed a bug with :ref:`CalculatePlaczek <algm-CalculatePlaczek>` algorithm for computing Placzek correction factors that fixed the previously implemented formula for transforming k to e, and the summation for second order Placezek corrections.
 - :ref:`ConvertAxesToRealSpace <algm-ConvertAxesToRealSpace>` no longer crashes Mantid if using an invalid file.
+- Calculation of log binning factor in :ref:`DiffractionFocussing <algm-DiffractionFocussing>` adjusted to remove a small discrepancy between total x range before\after rebinning.
```

Comment:
I think this backslash here needs escaping, as it's not appearing in the built docs. Or just a forward slash
```suggestion
- Calculation of log binning factor in :ref:`DiffractionFocussing <algm-DiffractionFocussing>` adjusted to remove a small discrepancy between total x range before\\after rebinning.
```
```suggestion
- Calculation of log binning factor in :ref:`DiffractionFocussing <algm-DiffractionFocussing>` adjusted to remove a small discrepancy between total x range before/after rebinning.
```